lua/lspsaga/diagnostic/show.lua

```diff
@@ -232,14 +232,24 @@ function sd:toggle_or_jump(entrys_list)
     end
     api.nvim_win_close(self.winid, true)
     local ln, col, bn = unpack(vim.split(info, ':'))
-    local wins = fn.win_findbuf(tonumber(bn))
+    local rownum = tonumber(ln)
+    local colnum = tonumber(col)
+    local bufnum = tonumber(bn)
+    if rownum == nil or colnum == nil or bufnum == nil or math.floor(rownum) ~= rownum or math.floor(colnum) ~= colnum or math.floor(bufnum) ~= bufnum then
```

Comment:
does there is necessary ? why need this judgement ?

Reply:
for our situation is safe.  it's always integer

Reply:
Okay, I see what you mean. Take a look at the current code